addons/mail/static/src/components/message_tracking_value/message_tracking_value.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates>
+    <t t-name="mail.MessageTrackingValue" owl="1">
+        <t t-if="messageTrackingValue">
```

Comment:
You can remove this `t-if` since a recent change to OWL 2 that makes it more robust to state that is being provided from props. This will remove 1 indent level (due to technical reasons), which makes template easier to read as a result.